<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvrMgrPage</class>
 <widget class="QWidget" name="SvrMgrPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QPushButton" name="freshButton">
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="setBootButton">
     <property name="text">
      <string>更改开机启动</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="openSvrButton">
     <property name="text">
      <string>服务控制</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>服务管理</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="svrTableWidget">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>171</number>
        </attribute>
        <column>
         <property name="text">
          <string>服务名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>进程ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>服务描述</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>开机启动</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>运行状态</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>408</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="restartSvrButton">
     <property name="text">
      <string>重启服务</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
